toc/working-groups/cf-on-k8s.md

```diff
@@ -0,0 +1,52 @@
+# Cloud Foundry on Kubernetes: Working Group Charter
+
+## Mission
+
+Bring the ease and simplicity of the Cloud Foundry developer experience to Kubernetes.
+
+
+## Goals
+
+- End-user developers can deploy their applications through the same convenient flows provided by their existing Cloud Foundry environments, using their existing clients and tools (like the `cf` CLI or the CF Java client), relying on the same (V3) API.
+- End-user developers can take advantage of Cloud Foundry via Kubernetes-native APIs, using their existing tools and clients (e.g. `kubectl`, `client-go`).
+- End-user developers can smoothly migrate their workloads from their existing Cloud Foundry environments to Cloud Foundry on Kubernetes.
+- End-user operators have a robust way to install Cloud Foundry on their Kubernetes clusters.
```

Comment:
I'm wondering if we can specify more what "robust" means here. I think it could imply something fully production-ready like cf-deployment which we're not equipped to develop or maintain. I'm wondering if that's actually a Non-Goal and our goal here is similar to the cf-for-k8s strategy of providing a deployment optimized for local development that others in the community can productionalize (dunno if this is a word 😅 ) if they want to.

cc @emalm for thoughts

Reply:
Yeah, we called out in the [Vision doc](https://docs.google.com/document/d/1rG814raI5UfGUsF_Ycrr8hKQMo1RH9TRMxuvkgHSdLg/edit) that "community reference deployment of CF on K8s should be optimized for initial, lightweight usage and experimentation", which aligns more to the last goal statement. Maybe it would make sense to rephrase this as something like "Vendors can base their own CF distributions on the core CF on K8s components"?